Local Pods/GitHubAPI/GitHubAPI/V3PullRequestFilesRequest.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  V3PullRequestFilesRequest.swift
+//  GitHubAPI
+//
+//  Created by Ryan Nystrom on 3/4/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+public struct V3PullRequestFilesRequest: V3Request {
```

Comment:
What about namespacing the `V3`?

Reply:
That’d require a separate framework right?

Reply:
Oh sorry, I meant having them nested. Like `V3.PullRequestFileRequest`.

Reply:
Ooooh I see! That seems doable. And the benefit is we could have GQL requests if the same name without collisions?